front-end/src/app/shared/components/contact-lookup/contact-lookup.component.html

```diff
@@ -1,29 +1,55 @@
-<form [formGroup]="contactLookupForm">
-  <p-dropdown
-    styleClass="gray"
-    formControlName="selectedContactType"
-    [options]="contactTypeOptions"
-    optionLabel="name"
-    optionValue="code">
-  </p-dropdown>
-  <p-dropdown 
-    #lookupDropdown
-    class="mr-2"
-    (input)="onDropdownSearch($event)"
-    [editable]="true"
-    dropdownIcon="pi pi-search"
-    [options]="contactLookupList" 
-    formControlName="selectedContact"
-    placeholder="Search for a contact"
-    [group]="true"
-    (onChange)="onContactSelect($event)">
-  </p-dropdown>
-  <span class="mr-2">or</span> 
-  <button
-    pButton
-    pRipple
-    label="Create a new contact"
-    icon="pi pi-plus-circle"
-    class="p-button-info mr-2">
-  </button>
-</form>
\ No newline at end of file
+<div class="p-fluid">
+  <form [formGroup]="contactLookupForm">
+    <div class="grid">
+      <div class="col-2 pl-0 pr-0">
+        <p-dropdown
+          [inputId]="contactTypeInputId"
+          [formControl]="contactTypeFormControl"
+          [options]="contactTypeOptions"
+          optionLabel="name"
+          optionValue="code"
+          [readonly]="contactTypeReadOnly"
+          [styleClass]="contactTypeStyleClass">
+        </p-dropdown>
+      </div>
+      <div class="col-2 pl-0">
+        <p-autoComplete 
+          [style]="{'width':'325px'}" 
+          [inputStyle]="{'width':'325px'}" 
```

Comment:
What is fixing the width of the autocomplete explicitly fix? It's not working dynamically when the screen size changes? Could we use percentages rather than pixels?

Reply:
Originally I had to set this to be able to resize the autocomplete component.  I was able to remove it now as it seems using bootstrap grid fixed this issue.